.wiki/Configuration.md

```diff
@@ -0,0 +1,35 @@
+The main Artipie configuration files is `/etc/artipie/artipie.yml`. It contains server meta configuration, such as:
+ - `layout` - `flat` or `org` string
+ - `storage` - repositories definition storage config
+ - `credentials` - user credentials config
+
+Example: 
+```yaml
+# /etc/artipie/artipie.yml
+meta:
+  layout: org
+  storage:
+    type: fs
+    path: /tmp/artipie/data/my-docker
+  credentials:
+    type: file
+    path: _credentials.yml
+```
+
+Layout specifies the URI path layout for Artipie. In case of `flat`, Artipie provides repositories at first path level, e.g. `artipie.host/repo1`, `artipie.host/repo2`. Only Artipie administrator can manage repositories in flat layout, users can't create new repositories or delete by self. `org` layout provides hierarchy support for repositories: URI path has two parts for org layouts: `<org>/<repo>`, where `<org>` is oganisation name, and `<repo>` the name of repository, e.g. `artipie.host/artipie/maven` - `maven` repository of `artipie` organisation. In org layout, organisation may have a maintainer who can manage repositories and permissions within organisation, it can add, delete and edit repositories, add granular permissions for users for each repository.
```

Comment:
@g4s8 please
- surround `org` at the beginning of the last sentence with quotation marks
- 

> organisation may have a maintainer who can manage repositories and permissions within organisation, IT can add...

I do not think it is appropriate to say "it" here